in_toto/runlib.go

```diff
@@ -299,14 +309,71 @@ func InTotoRun(name string, runDir string, materialPaths []string, productPaths
 		Command:     cmdArgs,
 		Environment: map[string]interface{}{},
 	}
+
 	linkMb.Signatures = []Signature{}
 	// We use a new feature from Go1.13 here, to check the key struct.
 	// IsZero() will return True, if the key hasn't been initialized
+
 	// with other values than the default ones.
 	if !reflect.ValueOf(key).IsZero() {
 		if err := linkMb.Sign(key); err != nil {
 			return linkMb, err
 		}
 	}
+
+	return linkMb, nil
+}
+
+func InTotoRecordStart(name string, materialPaths []string, key Key, hashAlgorithms, gitignorePatterns []string, lStripPaths []string) (Metablock, error) {
```

Comment:
Please add a brief description in form of docstrings for InTotoRecordStart/Stop.